spec/api/entities/search_result_spec.rb

```diff
@@ -56,5 +56,17 @@
       let(:result) { {'disk_evidence'=>0, 'redetermination'=>0, 'fixed_fee'=>0, 'awaiting_written_reasons'=>0, 'cracked'=>0, 'trial'=>1, 'guilty_plea'=>0, 'graduated_fees'=>1, 'interim_fees'=>1, 'warrants'=>0, 'interim_disbursements'=>0, 'risk_based_bills'=>0} }
       it { is_expected.to eql result }
     end
+
+    context 'when passed a litigator Transfer fixed fee' do
```

Comment:
As a refactor at some point it would be good to create some sort of "factory" to generate these OpenStructs. or use fixtures perhaps.